disdrodb/L0/readers/FRANCE/parser_ACTRIS-CCRES.py

```diff
@@ -0,0 +1,316 @@
+
+    # Skip first row as columns names
+    reader_kwargs["header"] = None
+
+    # Define encoding
+    reader_kwargs["encoding"] = "latin1"
+
+    ##------------------------------------------------------------------------.
+    #### - Define facultative dataframe sanitizer function for L0 processing
+    # - Enable to deal with bad raw data files
+    # - Enable to standardize raw data files to L0 standards  (i.e. time to datetime)
+    df_sanitizer_fun = None
+
+    def df_sanitizer_fun(df, lazy=False):
+        # Import dask or pandas
+        if lazy:
+            import dask.dataframe as dd
+        else:
+            import pandas as dd
+
+        # Remove header columns
+        df = df[
+            ~df.eq(
+                "Date;Time;Intensity of precipitation (mm/h);Precipitation since start (mm);Radar reflectivity (dBz);MOR Visibility (m);Signal amplitude of Laserband;Number of detected particles;Temperature in sensor (°C);Heating current (A);Sensor voltage (V);Kinetic Energy;Snow intensity (mm/h);Weather code SYNOP WaWa;Weather code METAR/SPECI;Weather code NWS;Spectrum"
+            ).any(1)
+        ]
+
+        # Split into columns and assign name
+        df = df["TO_SPLIT"].str.split(";", expand=True, n=16)
+
+        column_names = [
+            "date",
+            "time_temp",
+            "rainfall_rate_32bit",
+            "precipitation_since_start_TO_DROP",
+            "reflectivity_32bit",
+            "mor_visibility",
+            "laser_amplitude",
+            "number_particles",
+            "sensor_temperature",
+            "sensor_heating_current",
+            "sensor_battery_voltage",
+            "rain_kinetic_energy",
+            "weather_code_synop_4680",
+            "weather_code_metar_4678",
+            "weather_code_nws",
+            "I_DONT_KNOW_WHAT_IS",
+            "raw_drop_number",
+        ]
+
+        df.columns = column_names
+
+        # Drop precipitation_since_start_TO_DROP and I_DONT_KNOW_WHAT_IS column
+        df = df.drop(
+            columns=["precipitation_since_start_TO_DROP", "I_DONT_KNOW_WHAT_IS"]
+        )
+
+        # Parse time
+        df["time"] = df["date"] + "-" + df["time_temp"]
+        df["time"] = dd.to_datetime(df["time"], format="%Y/%m/%d-%H:%M:%S")
+        df = df.drop(columns=["date", "time_temp"])
+
+        # Set NaN into <SPECTRUM>ZERO</SPECTRUM> in raw_drop_number
+        import numpy as np
+
+        df["raw_drop_number"] = df["raw_drop_number"].replace(
+            "<SPECTRUM>ZERO</SPECTRUM>", np.NaN
+        )
+
+        # Cannot implement dask for this loop, so only pandas for now
+        for i, v in df.iterrows():
+            # Check if v is NaN
+            if not v["raw_drop_number"] != v["raw_drop_number"]:
+
+                v["raw_drop_number"] = v["raw_drop_number"].replace("<SPECTRUM>", "")
+                v["raw_drop_number"] = v["raw_drop_number"].replace("</SPECTRUM>", "")
+
+                # Add 0 digits to raw_drop_number
```

Comment:
The code here below could be delegated to the inner processing code. 
I will work on that on Friday ;)

Reply:
ok